src/test/java/redis/clients/jedis/JedisClusterTest.java

```diff
@@ -199,6 +199,32 @@ public void testReadonlyAndReadwrite() throws Exception {
     nodeSlave2.flushDB();
   }
 
+  @Test
+  public void testReadFromReplicas() throws Exception {
+    node1.clusterMeet(LOCAL_IP, nodeInfoSlave2.getPort());
+    JedisClusterTestUtil.waitForClusterReady(node1, node2, node3, nodeSlave2);
+
+    for (String nodeInfo : node2.clusterNodes().split("\n")) {
+      if (nodeInfo.contains("myself")) {
+        nodeSlave2.clusterReplicate(nodeInfo.split(" ")[0]);
+        break;
+      }
+    }
+
+    DefaultJedisClientConfig READ_REPLICAS_CLIENT_CONFIG
+        = DefaultJedisClientConfig.builder().password("cluster").readOnlyForReplicas().build();
+    try (JedisCluster jedisCluster = new JedisCluster(nodeInfo1, READ_REPLICAS_CLIENT_CONFIG,
+        DEFAULT_REDIRECTIONS, DEFAULT_POOL_CONFIG)) {
+      assertEquals("OK", jedisCluster.set("test", "read-from-replicas"));
+
+      ClusterCommandObjects commandObjects = new ClusterCommandObjects();
+      assertEquals("read-from-replicas", jedisCluster.executeCommandToReplica(commandObjects.get("test")));
```

Comment:
Are we sure the replica contains `read-from-replicas`?

Reply:
I run the test case again on my local PC. This `executeCommandToReplica` does return `read-from-replicas`. 
Please let me know if I have any mistakes.

Reply:
I meant ensuring data being served from `nodeInfoSlave2`.